RUFAS/output_manager.py

```diff
@@ -351,13 +366,20 @@ def _load_txt_file_to_list(self, path: str) -> List[str]:
             If an error occurs while opening or reading the file.
 
         """
+        info_map = {"class": self.__class__.__name__,
+                    "function": self._load_txt_file_to_list.__name__,
+                    }
+        self.add_log("open_text_file", f"Attempting to open {path}.", info_map)
         try:
             with open(path) as text_file:
-                return text_file.read().splitlines()
+                list_of_elements = text_file.read().splitlines()
+                load_message = f"Successfully opened {path}, it contained {len(list_of_elements)} filter patterns."
```

Comment:
```suggestion
                load_message = f"Successfully opened {path} and read {len(list_of_elements)} lines."
```
This function is "_load_txt_file_to_list" which means here we don't what are these lines yet. We had the same thing in a previous PR that led to renaming this function to its current name.
Also, at this point, we have successfully read the content of the file, we should log that too. Hence changing 'contained' to 'read'.